package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nahmii-sdk",
-  "version": "3.0.0",
+  "version": "3.1.0",
```

Comment:
Why the minor bump? Has there actually been any change to the public signature of the DriipSettlement class?

Reply:
No. Yet the update is not a bug fix.

Reply:
If its not a change to the interface it should be just a patch version

Reply:
I upgraded accordingly